hw23/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HW23</title>
+    <link rel="stylesheet" href="./fonts/font.css">
+    <script type="module" src="./js/script.js"></script>
+    <link rel="stylesheet" href="./styles/style.css">
+</head>
+
+<body>
+    <main>
+        <div class="container">
+            <form name="registrate" class="wrapper">
+                <h1>Sign up</h1>
+                <div>
+                    <label for="username" class="form">Username:</label>
+                    <input type="text" id="username" class="form" name="enter-username"
+                        placeholder="Enter your username">
+                    <span class="message form" data-for="enter-username"></span>
+                </div>
+                <div>
+                    <label for="email" class="form">Email:</label>
```

Comment:
I guess the green border is a bit confusing (when input has errors)

<img width="375" alt="image" src="https://user-images.githubusercontent.com/24583373/225750620-7bfafa84-a759-4de0-8951-4dcfbf56aea1.png">